test/inspector/test-off-no-session.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper.js');
+
+function testStop(harness) {
+  harness.expectShutDown(42);
+}
+
+helper.startNodeForInspectorTest(testStop, '--inspect',
```

Comment:
No. That test does not wait for the session. The tests cover shutting down the inspector with or without the session.